### ISSUE TITLE
Extract grade from the exercises notebook

### ISSUE THREAD

**Issue by InesPessoa** [LDSSA/nb-grades-collector#6]
Right now, the grade extracted is zero by default. Include the actual grade in the endpoint, by implementing the function that extracts the grade.

**Comment by buedaswag** [comment]
yes! also, we need to make sure the student saves the notebook before submitting (adding a message requesting it should do it)

if you agree, feel free to close https://github.com/LDSSA/nb-grades-collector/issues/8

**Comment by InesPessoa** [comment]
@buedaswag This example already warns students to save before running the grading cell. The #8 is to replace this method with something more reliable.
